dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -137,6 +137,24 @@ ifeq (auto,$(IOTLAB_NODE))
   endif
 endif
 
+# Handle IOTLAB_NODE specified with comma separated form
+# saclay,m3,1 => m3-1.saclay.iot-lab.info)
+_COMMA := ,
```

Comment:
Can't you use already defined `comma` makefiles/utils/variables.mk

Reply:
Thanks! Didn't know that this already exists :) Addressed and directly squashed!